core/services/fluxmonitor/flux_monitor_test.go

```diff
@@ -158,10 +180,23 @@ func TestPollingDeviationChecker_PollIfEligible(t *testing.T) {
 			const reportableRoundID = 2
 			latestAnswerNoPrecision := test.latestAnswer * int64(math.Pow10(int(initr.InitiatorParams.Precision)))
 
+			var availableFunds *assets.Link
+			var paymentAmount *assets.Link
+			minPayment := store.Config.MinimumContractPayment()
+			if test.funded {
+				availableFunds = minPayment
+				paymentAmount = minPayment
+			} else {
+				availableFunds = assets.NewLink(1)
+				paymentAmount = minPayment
+			}
+
```

Comment:
Can we look into how o simplify the tests?  They are getting difficult to understand / manage

Reply:
Steve and I have talked about doing this in a later PR as we need to get this to the auditors as soon as possible.